ansible_collections/arista/avd/roles/eos_cli_config_gen/defaults/main/main.yml

```diff
@@ -4,3 +4,7 @@ avd_structured_config_file_format: "yml"
 
 # Input Variable Validation
 avd_data_validation_mode: "error"
+
+# Input variable file/dir permission
+avd_dir_mode: 0o775
+avd_file_mode: 0o664
```

Comment:
We have a mix of where defaults are given. Some with filters, some here. We should align the all. @gmuloc thoughts?